### DISCUSSION TITLE
Capacity error

### DISCUSSION THREAD

**Discussion by RobinBluehub** [pybamm-team/PyBaMM#2159] [Q&A] [answered]
Why do I use the "Chen2020" battery parameter set for battery aging experiments, its nominal capacity is 5Ah, but the capacity decline starts from 5.127Ah, why is there an extra 0.127Ah?

**Comment by RobinBluehub** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/103869258/177685352-492f0a32-1bd5-40cf-af40-a3b55cef27a3.png)

**Reply by brosaplanella** [reply]
Which variable are you plotting for the capacity? Note that 5Ah is the nominal capacity (which is typically defined at a given C-rate) and the theoretical capacity (i.e. theoretical discharge capacity at an infinitely small C-rate) is usually higher.

**Reply by RobinBluehub** [reply]
Capacity

**Reply by RobinBluehub** [reply]
solution1.summary_variables["Capacity [A.h]"]

**Reply by brosaplanella** [reply]
Yes, that one is the theoretical one so it is higher.

**Reply by RobinBluehub** [reply]
In addition, I encountered another problem, and the solution could not be saved. 
solution=simulation.solve(starting_solution=solution_ini)
solution.save('DFN.pkl') 
is this command wrong?

**Reply by brosaplanella** [reply]
What error message are you getting?

**Reply by RobinBluehub** [reply]
The saved file is not displayed, that is, it is not saved without success.

**Reply by brosaplanella** [reply]
Might be an issue with the path... Can you specify an absolute path to check if it saves it where you want?

**Reply by RobinBluehub** [reply]
How to make an absolute path while keeping the complete data? Is there any documentation to refer to?

**Reply by brosaplanella** [reply]
This website might be a good introduction to paths: https://courses.cs.washington.edu/courses/cse140/13wi/file-interaction.html

Watch out that Windows has a different syntax to Linux and MacOs.
